list-publications/src/main/java/no/unit/nva/publication/query/PublishedPublicationsResponse.java

```diff
@@ -0,0 +1,17 @@
+package no.unit.nva.publication.query;
+
+import java.util.List;
+import no.unit.nva.publication.model.PublicationSummary;
+
+public class PublishedPublicationsResponse {
```

Comment:
Du kan også extende ArrayList for å returnere som JSON array

Reply:
Extending Arraylist for simplified response class